azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecServerModule.java

```diff
@@ -30,11 +37,49 @@
  */
 public class AzkabanExecServerModule extends AbstractModule {
 
+  private static final Logger logger = Logger.getLogger(AzkabanExecServerModule.class);
+  private static final String EVENT_REPORTER_CLASS_PARAM = "event.reporter.class";
+  private static final String EVENT_REPORTER_ENABLED = "event.reporter.enabled";
```

Comment:
Why isn't it prefixed with event.reporting like other configs?

Reply:
Same as previous comment.

Reply:
done.